matrix_client/room.py

```diff
@@ -469,11 +497,11 @@ def get_joined_members(self):
             {user_id: {"displayname": str or None}}: Dictionary of joined members.
         """
         response = self.client.api.get_room_members(self.room_id)
-        rtn = {
+        rtn = collections.OrderedDict({
```

Comment:
This ordering is important only if `/rooms/{room_id}/members` returns the
membership list in the order of membership events, so that we can select the
correct users as specified in
https://matrix.org/docs/spec/client_server/r0.2.0.html#calculating-the-display-name-for-a-room

Otherwise, we can drop the use of the OrderedDict entirely since I don't
think that endpoint gives joined members in any specific order. Does the
endpoint always give the order we're after?

Reply:
i removed the `OrderedDict` and the old tests